examples/servokit_all_servos_sequential.py

```diff
@@ -0,0 +1,15 @@
+"""Example that iterates through a servo on every channel, sets each to 180 and then back to 0."""
+import time
+from adafruit_servokit import ServoKit
+
+# Set channels to the number of servo channels on your kit.
+# 8 for FeatherWing, 16 for Shield/HAT/Bonnet.
+kit = ServoKit(channels=8)
+
+# Change range to the number of servos connected.
+# They must be on channels within the specified range!
+for i in range(8):
```

Comment:
Could you use `range(len(kit.servo))` here? Then you can drop the warning note...

https://github.com/adafruit/Adafruit_CircuitPython_ServoKit/blob/master/adafruit_servokit.py#L157

Reply:
@sommersoft The suggestion works! However, only makes sense if you're working with the exact number of servos as there are channels on the board. So if you connect 4, and say `range(len(kit.servo))` it'll work, but it's sending data to unused channels. 

Maybe it doesn't matter?

Reply:
> Maybe it doesn't matter?

It shouldn't, except to add time to overall loop. 

Like I said, its an option, but guards against any `out of range` occurrences. If you'd rather leave it as is, that's ok too.